README.md

```diff
@@ -36,3 +36,24 @@ Flags:
 
 Use "vault-unsealer [command] --help" for more information about a command.
 ```
+
+## How to setup vault-unsealer via AWS KMS and SSM
+
+[Instruction on existing and new vaults for unsealing vault using KMS and SSM](docs/aws-kms-ssm.md)
+
+## Build from source
+
+```bash
+git clone https://github.com/jetstack/vault-unsealer.git
+```
+
+```bash
+go get github.com/jetstack/vault-unsealerc
+make -C $(go env GOPATH)/src/github.com/jetstack/vault-unsealer build
+```
```

Comment:
go get is also doing the clone from the code block just before

Also there is now a small typo:
`go get github.com/jetstack/vault-unsealerc` -> `go get github.com/jetstack/vault-unsealer`

Reply:
cool yes i missed that section...

anyhow fixed round++